### DISCUSSION TITLE
Why nodeless?

### DISCUSSION THREAD

**Discussion by TheoOliveira** [gopi-suvanam/scribbler#26] [General]
I found this amazing project, which i intent to start contributing but first i'd like to know if you intent to make a cli and make like a node package following the same pattern python notebook has. I was amazed when i found out but I was looking for how to install using yarn and to my surprise nodeless. :)

**Comment by gopi-suvanam** [comment]
hi,
thanks for the comment..
There are a couple of reasons for not making the tool into a node package.. a few being - I don't want node_modules bloating up.. right now all files are static and no unnecessary modules are installed.. being static all files can be directly rendered from a static file server so people without node experience can also download and run the tool/ or directly host from github pages by just forking the repo.. as the files are static files, the source is the latest build - there is no need for separate build/deploy cycle..

to load it using CLI.. you can use these commands:
```
git clone https://github.com/gopi-suvanam/scribbler
cd scribbler
npx serve 
```
(I use `python -m SimpleHTTPServer`)

I'm open to putting it on npm also provided that does not interfere with the static nature of the project.. open to ideas!!  

Can you let me know what specific areas/issues you want to contribute to?
